src/Web/Larceny.hs

```diff
@@ -257,19 +260,36 @@ subs = M.fromList . map (\(x, y) -> (Blank x, y))
 -- | A plain text fill.
 --
 -- @
--- textFill "This text will be displayed in place of the blank"
+-- textFill "This text will be escaped and displayed in place of the blank"
 -- @
 textFill :: Text -> Fill s
 textFill t = textFill' (return t)
```

Comment:
It's very dangerous and it will break everything. 😈 

I just think it's waaay better to have the safer, more secure thing be the default, so it's worth the trouble. Which I think will be minimal -- I'll search and replace all the `textFill` in Offset with `rawTextFill`, which is definitely correct. In Wayland, almost all the `textFill`s used to be Interpreted Heist `textSplice`s that were and should be escaped. The main exception is the more complicated stuff in dynamic templates, that's the only part that should need changed.